docs/UserGuide.md

```diff
@@ -296,50 +296,44 @@ _Delivery on: 1st October 2023_
 
 _Details coming soon..._
 
-### Update delivery status and date
+### Update delivery status
 
-- Mark delivery as complete
-- Mark delivery as pending
-- Change date of delivery
+Changes the status of a specified delivery
 
-#### Mark delivery as complete
+**Format:** `delivery status STATUS CUSTOMER_ID`
 
-**Format**: `delivery complete DELIVERY_ID`
-
-**Example**: `delivery complete 1001`
+**Example:** `delivery status completed 2`
 
 **Accepted Values:**
 
-_DELIVERY_ID:_ Integer
-
-**Command succeeds:** _Delivery [1001] Gabriel’s Milk marked as complete!_
-
-**Command failed (delivery_id missing):** _Please specify a delivery id!_
-
-**Command failed (delivery_id not in database):** _This delivery does not seem to exist!_
-
-**Command failed (delivery already complete):** _This delivery is already marked as complete._
-
-#### Mark delivery as pending
+_STATUS:_ Either `CREATED`/`SHIPPED`/`COMPLETED`/`CANCELLED`
 
-**Format:** `delivery pending DELIVERY_ID`
+_CUSTOMER_ID:_ Integer
 
-**Example:** `delivery pending 1001`
+**Command succeeds:**
 
-**Accepted Values:**
+_Edited Delivery: [2] milk_
+_COMPLETED_<br />
+_Customer: Alex Yeoh_<br />
+_Customer Id: 1_<br />
+_Ordered On: 2023-10-21_<br />
+_Delivered On: 2023-12-03_
 
-_DELIVERY_ID_: Integer
+**Command fails (invalid_status):** _Delivery Status should be one of CREATED, SHIPPED, COMPLETED, CANCELLED_
 
-**Command succeeds:** _Delivery [1001] Gabriel’s Milk marked as pending!_
+**Command fails (invalid_index):** _The delivery index provided is invalid_
 
-**Command failed (delivery_id missing):** _Please specify a delivery id!_
+**Command fails (missing_fields):** 
 
-**Command failed (delivery_id not in database):** _This delivery does not seem to exist!_
-
-**Command failed (delivery already pending):** _This delivery is already marked as pending._
+_Invalid command format!_<br />
+_delivery status: Edits the status of the delivery identified by the ID of the delivery. Existing status will be overwritten by the input status._<br />
+_Parameters: STATUS (must be one of CREATED/SHIPPED/COMPLETED/CANCELLED) ID (must be a integer representing a valid ID)_<br />
+_Example: delivery status COMPLETED 1_<br />
 
 #### Change date of delivery
```

Comment:
We should classify this under delivery edit command instead of leaving it separate